support/ingress_test.go

```diff
@@ -0,0 +1,47 @@
+package support
+
+import (
+	"testing"
+
+	"github.com/onsi/gomega"
+
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+func NewFakeKubeClientForIngress(objects ...runtime.Object) *fake.Clientset {
```

Comment:
This function seems to be same as `NewFakeKubeClientWithObjects`, IMHO it would be good to reuse the existing one (and possibly move it some dedicated file to keep it on one place independent on specific test files).

Reply:
I kept this function in a separate file and reused it other files